src/xpk/commands/info.py

```diff
@@ -0,0 +1,240 @@
+"""
+Copyright 2024 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+from ..utils import xpk_exit, xpk_print
+from ..core.kueue import verify_kueuectl_installation
+from .cluster import set_cluster_command
+from ..core.commands import (
+    run_command_for_value,
+)
+from ..core.core import (
+    add_zone_and_project,
+)
+import json
+from tabulate import tabulate
+from argparse import Namespace
+
+table_fmt = 'plain'
+
+
+def prepare_kueuectl(args: Namespace) -> None:
```

Comment:
WDYT about moving this function to `core/kueue.py`? It seems like it could be used outside of the `xpk info` command, similarly to `verify_kueuectl_installation`